pkg/config/config.go

```diff
@@ -300,8 +300,9 @@ func (c *ServerConfig) ValidateAndAdjust() error {
 
 	if c.Sorter == nil {
 		c.Sorter = defaultServerConfig.Sorter
```

Comment:
I think it's not good to assign defaultServerConfig.Sorter to other point which will be out of control and cause data race.  

something like c.Sorter = GetDefaultServerConfig().Sorter or c.Sorter = defaultServerConfig.Sorter.Clone() is better

Reply:
line 334:  c.KVClient = defaultServerConfig.KVClient 
do other fields in ServerConfig has the same problem?

Reply:
BTW,  pr body need to be more specified